security_drone_user_app/test/thief_bloc_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/cupertino.dart';
+import 'package:security_drone_user_app/config.dart';
+import 'package:security_drone_user_app/data/models/lat_lng_point.dart';
+import 'package:security_drone_user_app/data/models/thief_entry.dart';
+import 'package:security_drone_user_app/logic/thief_page_bloc.dart';
+import 'package:bloc_test/bloc_test.dart';
+import 'dart:io';
+
+void cleanFile() {
+  final file = new File(testsOutputFile);
+  file.writeAsStringSync('');
+}
+
+void main() {
+
+  var point = LatLngPoint(35.12, 34.11);
+  var entry = ThiefEntry(Image.asset("assets/images/thief.jpg"), DateTime.now(), point);
+
+  // say the we are testing
```

Comment:
change the to that

Reply:
לא מובנת לי הערה

Reply:
the comment says "say the we are testing",
you should write instead "say that we are testing"